pkgs/by-name/ho/horcrux/package.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "horcrux";
+  version = "0-unstable-2023-09-19";
+
+  src = fetchFromGitHub {
+    owner = "jesseduffield";
+    repo = "horcrux";
+    rev = "5e848abcca49a7ad359f5a24ef4ca7e0eda80889";
+    hash = "sha256-YOu3qJadfyA6MKW8OFLr0pFjGMOgCGie2f8VbG79IY0=";
+  };
+
+  vendorHash = null;
```

Comment:
I haven't come across `vendorHash` being see that many (~290) packages use it. What are reasons for `vendorHash` being `null` in general and what is the motivation to do so for this package?

Reply:
https://ryantm.github.io/nixpkgs/languages-frameworks/go/

> vendorHash: is the hash of the output of the intermediate fetcher derivation.
> vendorHash can also be set to null. In that case, rather than fetching the dependencies and vendoring them, the dependencies vendored in the source repo will be used.

It is also used in cases where the package is pure, not requiring extra dependencies.

Reply:
Thank you for providing additional context and the source to it, very helpful, @AndersonTorres 🙏